packages/terra-form-select/src/single/Menu.jsx

```diff
@@ -161,8 +161,9 @@ class Menu extends React.Component {
     this.clearSearch();
     this.clearScrollTimeout();
     if (this.state.closedViaKeyEvent) {
-      this.props.select.focus();
-      if (SharedUtil.isSafari()) {
+      if (!SharedUtil.isSafari()) {
```

Comment:
Can we add unit tests for both this condition and the one [below](https://github.com/cerner/terra-core/pull/3811/files#diff-baf4aa45e008c306518e7aab26fc079bd1ea7bdfbac075231c6593781384d140R264)?